Module 10 - Javascript and Ajax/pages/DeleteStudent.page

```diff
@@ -0,0 +1,52 @@
+<apex:page standardController="Student__c" recordSetVar="studentDetails" extensions="DeleteStudentController" showHeader="false" >
```

Comment:
this page should not have any extension. every thing should be handled in the js only